.eslintrc

```diff
@@ -8,7 +8,8 @@
   ],
   "rules": {
     "guard-for-in": 0,
-    "no-inline-comments": 0
+    "no-inline-comments": 0,
+    "no-continue": 0
```

Comment:
This lint rule might be better to disable on a case by case basis. Avoiding `continue` in loops is a good practice.